.github/workflows/validate.yaml

```diff
@@ -0,0 +1,33 @@
+---
```

Comment:
Running workflows can be seen at my fork: https://github.com/cardil/serverless-operator/actions

After merge they will be in this repo